### ISSUE TITLE
No sound on OS X with v0.6.5

### ISSUE THREAD

**Issue by nicroto** [Soundnode/soundnode-app#1057]
I successfully authenticate, but nothing can be played. If I try to skip ahead to the middle of a song - an error is displayed "Oh, something went wrong!" or something similar.

**Comment by nicroto** [comment]
Looking at the growing number of thumbs-up, I guess I am not the only one having this issue. Does anyone on the dev team have it working on OS X?

**Comment by jakejarrett** [comment]
Most likely the same issue as #802 

You might be able to mitigate the issue by changing your API key on 7.0.0 release (Github releases tab)

**Comment by nicroto** [comment]
@jakejarrett thanks for the reply.

I think that's it, yeah. But SoundCloud doesn't accept applications for new API keys. Any alternative solutions?

**Comment by jakejarrett** [comment]
You're able to get one of the SoundCloud API keys by viewing network requests on SoundCloud website (devtools)

**Comment by nicroto** [comment]
Hm... OK. Didn't know I could do it this way.

The issue I am facing now is that with latest version, the right click is disabled everywhere (including in Chrome DevTools) and so are pretty much all keyboard shortcuts (Cmd + V; Cmd + Q; etc.). I think I saw somewhere you are migrating to a new shell so those, I am guessing, are issues with the transition (haven't set those up, likely)? I think the issue comes from the native menu - Quit isn't available as a command so no Cmd + Q, Edit actions aren't either so no Cmd + C/V, etc.

**Comment by jakejarrett** [comment]
Not soundnode's devtools, You can just go to soundcloud, stream one of the songs and see the outgoing request and re-use the `client_id` as your api key.

**Comment by nicroto** [comment]
Sure, that's what I did.

I am saying that since I am unable to copy the text into Soundnode, I tried to copy it directly in the DOM of the app through the devtools but copy/paste command are not enabled even there.

**Comment by jakejarrett** [comment]
You can change the API key in the settings panel on version `7.0.0` or higher (`Mod/CMD/Ctrl + ,`)

**Comment by nicroto** [comment]
Bro... I can't. That's what I am saying. I am testing on OS X 10.12.6, so maybe you don't support that?

As I showed you on the screenshot - pretty much all window commands are missing, including the ones dealing with the clipboard (the Edit menu).

**Comment by nicroto** [comment]
Yes it does. I can open the settings and I see the key input field, but I can't paste into it... Ah, I'll just type in the client_id... Nope, this doesn't work either - after I type it in, it doesn't seem to remember it and on restart it simply loads the previous one.

**Comment by jakejarrett** [comment]
First issue is currently awaiting to be merged #1039 & the saving one has been merged but not released yet

**Comment by jamesdh** [comment]
You guys realize that this combination of issues (1. unable to paste, 2. having to dig around and find your client_id in a header) basically make all of the other great work you've done...meaningless? 

Just gave this app a try. The version available on Homebrew is outdated and doesn't even allow setting the API key. Eventually found this issue. Uninstalled, reinstalled from GitHub Releases. Unable to login with my password manager because the login screen doesn't accept pastes. Have to manually type in gibberish. Have to manually type in again because I apparently mistyped it. Finally get to Settings screen. Can't paste in the client ID. Type it in. Type it in again because I again mistyped one of the 32 chars. Finally get it in, go back to main screen, hit play. A bit of quirky behavior, but then music finally starts playing. 

I guess what I'm saying is that, considering the amount of great work that appears to have gone into other parts of this app, it seems pretty crazy that there are these two issues still outstanding which basically render it useless for the vast majority users. Most people won't have the knowhow to get this working.
